subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/SubscriptionService.kt

```diff
@@ -162,6 +164,21 @@ class SubscriptionService(
             }
     }
 
+    fun getContextsById(id: URI): Mono<List<String>> {
```

Comment:
the name of the function is not really good. indeed, you are getting the contexts for a subscription (not by id of contexts)

Reply:
maybe `getContextsBySubscriptionId`?